tools/Rdata-reader/rdata_extractor.xml

```diff
@@ -1,25 +1,28 @@
 <tool id="Rdata_exctract" name="Rdata extractor" version="0.1">
-  <command detect_errors="exit_code"><![CDATA[
+    <requirements>
+       <!-- <requirement type="package" version="3.2.1">R</requirement>-->
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
 	Rscript $__tool_directory__/rdata_extractor.R $rdata $selectedattributes $summary
```

Comment:
the tool_directory should also be singel quoted

Reply:
$rdata as well. Essentially all text params, all data params and all paths should be single-quoted